ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template3.kt

```diff
@@ -67,6 +69,17 @@ internal fun Template3(
         ) {
             Template3MainContent(state)
         }
+        Column(
+            modifier = Modifier.align(Alignment.CenterHorizontally),
```

Comment:
Hmm could we apply this alignment to the parent column? That way we avoid this column

Reply:
That doesn't work because I'd have to use `ColumnScope`.